script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -1703,7 +1709,7 @@ That's why she wants to die――so she can finally be at peace.
 [sha:acbf891d879a80ccd318bc5f7903d0715015f0a7]{
 -- Page 10028, Offset 33864.
 -- 「違う……そんなのは、違う」
-// Assuming this is more of a 'you shouldn't kill yourself' than a 'no, you are incorrect'.
+// Assuming this is more of a 'you shouldn't kill yourself' / dying won't bring you peace than a 'no, you are incorrect'.
 "No... You're... making a mistake."
```

Comment:
I still feel as though this one is hard to get a grasp on. The thing we have makes sense in context of the translation, so it works, but I get the feeling that it's not quite right. Well, it's beyond me as of now.